### PR TITLE
Improve MathML for extensible arrows

### PR DESCRIPTION
**Pull request by ronkok**
Add padding to the MathML of extensible arrows. They should be a little longer than their subject matter.

### CODE REVIEW

File: src/functions/arrow.js [hunk 1]
```diff
@@ -119,9 +129,15 @@ defineFunction({
...
+            lowerNode = paddedNode(lowerNode);
             node = new mathMLTree.MathNode("munder", [arrowNode, lowerNode]);
         } else {
-            node = new mathMLTree.MathNode("mover", [arrowNode]);
+            // This should never happen.
+            // Parser.js throws an error if there is no argument.
+            node = new mathMLTree.MathNode("mpadded", []);
+            node.setAttribute("width", "+0.6em");
+            node.setAttribute("lspace", "0.3em");
+            node = new mathMLTree.MathNode("mover", [arrowNode, node]);
```
Comment (ylemkimon) [inline src/functions/arrow.js]:
```js
node = paddedNode();
```

File: src/functions/arrow.js [hunk 2]
```diff
@@ -108,9 +116,11 @@ defineFunction({
         let lowerNode;
 
         if (group.body) {
-            const upperNode = mml.buildGroup(group.body, options);
+            let upperNode = mml.buildGroup(group.body, options);
+            upperNode = paddedNode(upperNode);
```
Comment (ylemkimon) [inline src/functions/arrow.js]: I think function calls can be nest, as they're straightforward:
```js
const upperNode = paddedNode(mml.buildGroup(group.body, options));
```

File: src/functions/arrow.js (hunk 1 above)
Comment (ronkok) [reply src/functions/arrow.js]: Done, with a modification as needed to satisfy `flow`.

File: src/functions/arrow.js (hunk 2 above)
Comment (ronkok) [reply src/functions/arrow.js]: Done.

File: src/functions/arrow.js [hunk 3]
```diff
@@ -105,23 +115,25 @@ defineFunction({
...
+                const lowerNode = paddedNode(mml.buildGroup(group.below, options));
                 node = new mathMLTree.MathNode(
                     "munderover", [arrowNode, lowerNode, upperNode]
                 );
             } else {
                 node = new mathMLTree.MathNode("mover", [arrowNode, upperNode]);
             }
         } else if (group.below) {
-            lowerNode = mml.buildGroup(group.below, options);
+            const lowerNode = paddedNode( mml.buildGroup(group.below, options));
```
Comment (ylemkimon) [inline src/functions/arrow.js]:
```suggestion
            const lowerNode = paddedNode(mml.buildGroup(group.below, options));
```

File: src/functions/arrow.js [hunk 4]
```diff
@@ -9,6 +9,16 @@ import * as mml from "../buildMathML";
 
 import type {ParseNode} from "../parseNode";
 
+// Helper function
+const paddedNode = group => {
+    const node = (group)
```
Comment (ylemkimon) [inline src/functions/arrow.js]:
```js
    const node = new mathMLTree.MathNode("mpadded", group ? [group] : []);
```

File: src/functions/arrow.js (hunk 3 above)
Comment (ronkok) [reply src/functions/arrow.js]: Done.

File: src/functions/arrow.js (hunk 4 above)
Comment (ronkok) [reply src/functions/arrow.js]: Done.

**Pull request merged**